rtl/include/eclhelper.hpp

```diff
@@ -38,7 +38,7 @@ if the supplied pointer was not from the roxiemem heap. Usually an OwnedRoxieStr
 #include "unicode/utf.h"
 #else
 #ifdef _WIN32
-typedef wchar_t UChar;
+typedef char16_t UChar;
```

Comment:
Will this change any signatures of (non-virtual) API fucntions that might result in queries not being compatible? I suppose we probably don't mind so much if it's only windows though?

Reply:
@richardkchapman Yes it will.  But I think that is ok for windows only.